### ISSUE TITLE
<!--nextpage--> does not link to an amp page

### ISSUE THREAD

**Issue by brianlayman** [Automattic/amp-wp#292]
This can be adjusted in the theme of course, but it probably should be added to the amp plugin with something similar to a trailingampit() function.

This is what I use:
/**
 * Ensures the passed text, presumably an url, ends with '/amp/'
 */
if ( !function_exists( 'trailingampit' ) ) {
	function trailingampit( $string ) {
		return untrailingampit( $string ) . 'amp/';
	}
}

/**
 * Ensures the passed text, presumably an url, does not end with 'amp/' or 'amp'
 */
if ( !function_exists( 'untrailingampit' ) ) {
	function untrailingampit( $str ) {
		$result = preg_replace( '/(amp|amp\/)$/', '', $str );
		return $result;
	}
}

**Comment by mjangda** [comment]
@brianlayman I think some of your code/issue/formatting may have been stripped. Can you update the issue to fix?

I'm also not sure I understand what the issue is. Can you share repro steps?

**Comment by mjangda** [comment]
Closing due to lack of information. Please re-open if this continues to be an issue.

**Comment by brianlayman** [comment]
Sorry for the lack of info and stripped content.

So here's what's happening.  The  tag in a post creates a link inside the post to goto the next bunch of content.  However, that link to the page 2 of the content on amp articles is currently taking you out of an AMP context because the link is to the normal page 2 of the content. Once on an amp page all previous and next links SHOULD keep you in an AMP context.  This will eventually apply to archive pages too. Though google doesn't focus on these, you do go through them when browsing a site and once in AMP mode, the visitor really shouldn't drop out of it through no intentional action of their own.

So I created a function modeled after trailingslashit called traillingampit.

Here is a gist of the code I created:
https://gist.github.com/brianlayman/b43e82e61b2fc67829cab6565684d839

It would be good if the plugin could handle this and people didn't need to do it in the themes..
